besu/src/test/java/org/hyperledger/besu/controller/AbstractBftBesuControllerBuilderTest.java

```diff
@@ -106,6 +107,18 @@ public void setup() throws JsonProcessingException {
     lenient().when(genesisConfig.getNonce()).thenReturn(Long.toHexString(1));
     lenient().when(genesisConfig.getConfigOptions()).thenReturn(genesisConfigOptions);
     lenient().when(genesisConfigOptions.getCheckpointOptions()).thenReturn(checkpointConfigOptions);
+    lenient()
+        .when(genesisConfigOptions.getDepositContractAddress())
+        .thenReturn(
+            Optional.of(Address.fromHexString("0x00000000219ab540356cbb839cbe05303d7705fa")));
+    lenient()
+        .when(genesisConfigOptions.getWithdrawalRequestContractAddress())
+        .thenReturn(
+            Optional.of(Address.fromHexString("0x00000961ef480eb55e80d19ad83579a64c007002")));
+    lenient()
+        .when(genesisConfigOptions.getConsolidationRequestContractAddress())
+        .thenReturn(
+            Optional.of(Address.fromHexString("0x0000bbddc7ce488642fb579f8b00f3a590007251")));
```

Comment:
it feels wrong to shoe-horn in mainnet system contract addresses for a consensus mechanism test that shouldn't otherwise ever have these contracts 🤔 .  What does the deposit, withdrawal, and consolidation system contract mean in the context of a *Bft consensus chain?

Reply:
Ah yes, and this is no longer needed since I added the conditionals around adding the milestones to the schedule. 

Removed.